### PR TITLE
Fix pycodestyle global_config

### PR DESCRIPTION
**Pull request by datamel**
Small problem, fixes pycodestyle in global config.
Should help actions style pass.

This is a small change with no associated Issue.

**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Does not need tests (why?).  Small pycodestyle fix
- [x] No change log entry required (invisible to users).
- [x] No documentation update required.

### CODE REVIEW

File: cylc/flow/cfgspec/globalcfg.py [hunk 1]
```diff
@@ -344,7 +345,8 @@
             ''')
             Conf('retrieve job logs', VDR.V_BOOLEAN, desc='''
                 Global default for
-                :cylc:conf:`suite.rc[runtime][<namespace>][remote]retrieve job logs`.
+                :cylc:conf:`suite.rc[runtime][<namespace>][remote]retrieve job
+                logs`.
```
Comment (oliver-sanders) [inline cylc/flow/cfgspec/globalcfg.py:349]: Damn, I remember this was something I was going to go back to later.

When you build the docs does this reference still work? The cylc domain might possibly need some tweaking to let it pass, or I might have already done this and forgotten.

Comment (datamel) [reply cylc/flow/cfgspec/globalcfg.py:349]: I have built it and I think it works as intended.

Comment (oliver-sanders) [reply cylc/flow/cfgspec/globalcfg.py:349]:
```
Exception occurred:
  File "/Users/oliver/cylc-sphinx-extensions/cylc/sphinx_ext/cylc_lang/domains.py", line 118, in tokenise
    raise ValueError(f'Not a valid namespace "{namespace_string}"')
ValueError: Not a valid namespace "suite.rc[runtime][<namespace>][remote]retrieve job
logs"
```

Comment (oliver-sanders) [reply cylc/flow/cfgspec/globalcfg.py:349]: Simple fix: https://github.com/cylc/cylc-sphinx-extensions/pull/25

**Comment by oliver-sanders** [comment]
Ok to let cylc-doc break for a short while.

File: cylc/flow/cfgspec/globalcfg.py (hunk 1 above)
Comment (datamel) [reply cylc/flow/cfgspec/globalcfg.py:349]: hmmm weird I didn't get that error

Comment (oliver-sanders) [reply cylc/flow/cfgspec/globalcfg.py:349]: hmmm, are all three branches upto date.

Comment (datamel) [reply cylc/flow/cfgspec/globalcfg.py:349]: :) oopsie, hadn't updated master on cylc sphinx extensions, I'm such a lemon sometimes!

Comment (datamel) [reply cylc/flow/cfgspec/globalcfg.py:349]: I double checked this, managed to get that failure. Now working well with other pr checked out.

**Pull request merged**